### PR TITLE
VS-815: Add Support for YNG to VQSR Lite

### PR DESCRIPTION
**Pull request by gbggrant**

### CODE REVIEW

**Comment by gbggrant** [comment]
**For VQSR Lite:**
Passing GvsCreateFilterSet run [here](https://app.terra.bio/#workspaces/allofus-drc-wgs-dev/NHGRI_AnVIL_3K_AoU_Development/job_history/a3b515eb-e7bc-4f29-b760-ca1d5e8f8e49)
Passing GvsExtractCallset run [here](https://app.terra.bio/#workspaces/allofus-drc-wgs-dev/NHGRI_AnVIL_3K_AoU_Development/job_history/a0db2735-c9dd-475d-9bb7-0f4a167b90a0)

**For VQSR Classic:**
Passing GvsCreateFilterSet run [here](https://app.terra.bio/#workspaces/allofus-drc-wgs-dev/NHGRI_AnVIL_3K_AoU_Development/job_history/611adf34-603b-45fe-8c2d-a982d8ae3c4b)
Passing GvsExtractCallset run [here](https://app.terra.bio/#workspaces/allofus-drc-wgs-dev/NHGRI_AnVIL_3K_AoU_Development/job_history/6e175c44-8aa0-433f-a9ca-9bfd16e1b06d)

File: scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl [hunk 1]
```diff
@@ -75,15 +78,16 @@ workflow GvsCalculatePrecisionAndSensitivity {
         output_basename = output_sample_basename
     }
 
-    call Add_AS_MAX_VQSLOD_ToVcf {
+    call Add_AS_MAX_VQS_Score_ToVcf {
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl:81]: is it VQS score? not VQSR? Doesn't the S for "score" ?

File: scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl [hunk 2]
```diff
@@ -1,5 +1,7 @@
 version 1.0
 
+# I am a comment.
+
```
Comment (rsasch) [inline scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl:4]: why is this here?

File: scripts/variantstore/wdl/GvsCreateFilterSet.wdl [hunk 3]
```diff
@@ -4,6 +4,8 @@ import "GvsWarpTasks.wdl" as Tasks
 import "GvsUtils.wdl" as Utils
 import "../../vcf_site_level_filtering_wdl/JointVcfFiltering.wdl" as VQSRLite
 
+# This is a reminder to set the priors back to non-zero in GvsWarpTasks before you merge this PR!!  TODO
+
```
Comment (rsasch) [inline scripts/variantstore/wdl/GvsCreateFilterSet.wdl:8]: Can this be done with a passed argument?

**Review by rsasch — COMMENTED** [review]
Do we have to worry about the changes to JointVcfFiltering.wdl and CreateFilteringFiles.java affecting people outside the GVS?

File: scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl (hunk 2 above)
Comment (gbggrant) [reply scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl:4]: sorry - this is temp debugging code. A stupid way I use to verify that the WDL I'm about to run in Terra has the change I just pushed. Will be removed shortly.

File: scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl (hunk 1 above)
Comment (gbggrant) [reply scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl:83]: Yeah - This label is very specific to this WDL and I was just looking for a generic term to use for lod and calibration_sensitivity. I'll change it up.

File: scripts/variantstore/wdl/GvsCreateFilterSet.wdl (hunk 3 above)
Comment (gbggrant) [reply scripts/variantstore/wdl/GvsCreateFilterSet.wdl:8]: Yeah, that'd be a good idea. I'm still working with Sam to make sure I run classic in the way he wants in order to remove YNG issues. So if what I've done here isn't the way (which I think is what's happening) I may remove this all - but I will try to parameterize it for future work.

File: scripts/vcf_site_level_filtering_wdl/JointVcfFiltering.wdl [hunk 4]
```diff
@@ -177,10 +181,11 @@ task ExtractVariantAnnotations {
 		File extracted_training_vcf = "~{basename}.~{mode}.vcf.gz"
 		File extracted_training_vcf_index = "~{basename}.~{mode}.vcf.gz.tbi"
 		Array[File] outputs = glob("~{basename}.~{mode}.*")
+		File monitoring_log = "monitoring.log"
 	}
 	runtime {
 		docker: gatk_docker
-		disks: "local-disk " + disk_size + " LOCAL"
```
Comment (gbggrant) [inline scripts/vcf_site_level_filtering_wdl/JointVcfFiltering.wdl:183]: Interesting thing here - 'LOCAL' calls for SSD in chunks of 375GB https://cloud.google.com/compute/docs/disks/#overview (as I read it). I switched to HDD for cost. I *don't* think performance took a big hit

**Comment by gbggrant** [comment]
> 

@rsasch I don't believe that CreateFilteringFiles.java is used anywhere else except in GVS. As for JointVcfFiltering.wdl, I really tried to avoid changing it, but the changes I had to make were for memory and disk. At this point the changes are in our branch (ah_var_store) and when we merge with main, we can deal with the PR then. But, I also know there's an updated version of this WDL that we are supposed to update to - that will presumably happen before we merge with main.

**Review by rsasch — APPROVED** [review]
Thanks for the explanation; assuming you implement the stuff you described 👍🏻 .
